prometheus/registry.go

```diff
@@ -114,6 +114,16 @@ func Register(m Collector) error {
 	return err
 }
 
+// RegisterAll works like Register but takes a variadic list of Collectors.
+func RegisterAll(m ...Collector) error {
+	for i := range m {
+		if err := Register(m[i]); err != nil {
+			return err
```

Comment:
What would you think about returning a descriptive error containing the exact argument that failed? Then the usage between the two functions gets really clean, since it's essentially the same as before, but variadic for both.